features/step_definitions/tag_steps.rb

```diff
@@ -374,10 +374,21 @@
   click_button("Save changes")
 end
 
+When "I assign the tag {string} to the fandom {string}" do |tag, fandom|
```

Comment:
Could we update/repurpose the existing step that adds the fandom to character tags instead, or is it important we visit the page instead of using `add_association`? 

https://github.com/otwcode/otwarchive/blob/6443729393955d8a75a4f4e86772428aabdc9191/features/step_definitions/tag_steps.rb#L87-L91